posydon/binary_evol/SN/profile_collapse.py

```diff
@@ -24,6 +24,40 @@
     'Aldo Batta <aldobatta@gmail.com>',
 ]
 
+def get_ejecta_element_mass_at_collapse(star, compact_object_mass, verbose):
+    # read star quantities
+    enclosed_mass_all = star.profile['mass'][::-1]*Mo  # cell outer total mass
```

Comment:
`Mo` should be replaced by `const.Msun`.

Reply:
I think, we need to go once more carefully through the unit changes to make sure all is as it should.

Reply:
I think that grams are not needed at all in the function in the end, so everything should be in Msun now